src/utils/sources-tree/tests/updateTree.spec.js

```diff
@@ -0,0 +1,34 @@
+import { updateTree } from "../index";
+import { createNode } from "../utils";
+
+describe("calls updateTree.js", () => {
+  it("updates source tree correclty", () => {
+    // set props
+    const props = {
+      debuggeeUrl: "blah"
+    };
+    props.sources = new Map();
+    props.sources.set("1", "str1");
+
+    // set nextProps
+    const nextProps = {
+      debuggeeUrl: "blah",
+      projectRoot: ""
+    };
+    nextProps.sources = new Map();
```

Comment:
it looks like this isnt an immutable map, but a plain js Map https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

This doesn't have a valueSeq method and is probably why the tests are failing!

you can fix this by importing immutable's "Map" and using that here